modules/cms/classes/Controller.php

```diff
@@ -488,11 +488,16 @@ protected function initTwigEnvironment()
 
         $useCache = !Config::get('cms.twigNoCache');
         $isDebugMode = Config::get('app.debug', false);
+        $strictVariables = Config::get('cms.enableTwigStrictVariables', false);
+        if ($strictVariables === null) {
```

Comment:
use `is_null()` please. In the first place, the performance difference between them is ridiculously slim, in the second place PHP7 actually reverses the performance difference. PHP5.6 `===` is ~100 nano seconds faster, PHP7 `is_null()` is ~5 nano seconds faster. We use `is_null` elsewhere in the code so just use it here too please.